guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md

```diff
@@ -298,6 +300,23 @@ Example:
 </arguments>
 ```
 
+### XML layout File merging ordering {#xml_Layout_Merging}
+
+**Module Files**
+Sorted according to their module priority (array index of module's position in app/etc/config.php). If their priority are equal, they are sorted according to their alphabetical priority.
+
+- Module base file loaded
+
+- Module Area files loaded
+
+**Theme files**
+
+- Layout file loaded
```

Comment:
Hi @speedy008. Thank you for the provided description. 
We need to make the explanation more transparent for the end-user. So we have something like the following list
```suggestion
- Module base layout file loaded
- Module area layout file loaded
- Theme layout file loaded
...
```

Also, there's one explanation missing regarding theme layouts from the inheritance perspective. Usually a theme extends another theme, so layouts from a parent theme also should be taken into account.

Reply:
Hi @speedy008. By the three dots in my example, I meant that the other contents of the list should follow the example, so we don't need the three dots in the list literally. 

To clarify my request, what needs to be fixed from your initial commit:
- Specify the layout source (i.e. module layout, theme layout)
- Add information about a parent theme layout loading order (it was missing in your initial commit)

Thank you

Reply:
I have added path information please review.

Reply:
The information about loading order of a child -> parent theme is missing. It's a very important part of the theme layout merging flow

Reply:
@speedy008 Can you confirm whether the loading order of a child -> parent theme is needed, and make those changes if necessary?

Reply:
@shrielenee The information I have added is include parent theme information as well.